docs/build/blast-tutorial-1.md

```diff
@@ -3,29 +3,18 @@ title: Create Counter project
 id: create-counter
 ---
 
-This short tutorial walks you through setting up your project and creating a simple counter that increments upon request using **Cudos Blast**.
+This short tutorial creates a simple counter that increments upon request using **Cudos Blast**.
 
-It assumes that you have installed **Cudos Blast** globally. 
+1. Check your in the 'counter project' you created in [Cudos Blast](/docs/build/blast-install-1.md).
```

Comment:
As noted, you didn't create a Blast project on that page. You should either restore the `blast init` instructions here, or add them to the Blast installation page.

Reply:
Done